readme.md

```diff
@@ -45,7 +45,7 @@ findCacheDir({name: 'unicorns'});
 
 ### findCacheDir(options?)
 
```

Comment:
The readme also needs to mention some of the use-cases for `CACHE_DIR`.

Reply:
Done!